notebooks/01-visualize-amounts-bar-plots.ipynb

```diff
@@ -0,0 +1,1384 @@
+{

```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        line_width=2.5,  # line width separating the bars
</pre><p>The line is an outline for the glyph, it's not really related directly to separating the bars though that is a side-effect. Maybe "increased line width to separate the bars" ? That keeps true about what <code>line_width</code> is technically, but speaks to the motivation for changing it.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/bokeh/dataviz-fundamentals/pull/19/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>